Gemfile

```diff
@@ -24,6 +24,8 @@ gem "simple_form"
 gem "title"
 gem "uglifier"
 gem "unicorn"
+gem "oj"
```

Comment:
Let's try to keep the list of gems alphabetized, it helps to find things and reduces the chance that people duplicate gems in the list -- or specify conflicting versions.